hbase-client/src/main/java/org/apache/hadoop/hbase/replication/ReplicationLoadSink.java

```diff
@@ -22,7 +22,6 @@
   private final long timestampStarted;
   private final long totalOpsProcessed;
 
-  // TODO: add the builder for this class
   @InterfaceAudience.Private
   public ReplicationLoadSink(long age, long timestamp, long timestampStarted,
```

Comment:
The constructor should be private when you use Builder pattern.

Reply:
Right, will change it to private in the latest commit.